src/Symfony/Component/Notifier/Bridge/Telegram/Tests/TelegramTransportTest.php

```diff
@@ -28,15 +28,17 @@ final class TelegramTransportTest extends TransportTestCase
 {
     private const FIXTURE_FILE = __DIR__.'/Fixtures/image.png';
 
-    public static function createTransport(?HttpClientInterface $client = null, ?string $channel = null): TelegramTransport
+    public static function createTransport(?HttpClientInterface $client = null, ?string $channel = null, bool $disableHttps = false): TelegramTransport
     {
-        return new TelegramTransport('token', $channel, $client ?? new MockHttpClient());
+        return new TelegramTransport('token', $channel, $client ?? new MockHttpClient(), null, $disableHttps);
```

Comment:
```suggestion
        return new TelegramTransport('token', $channel, $client ?? new MockHttpClient(), disableHttps: $disableHttps);
```

Reply:
Actually was gonna do the same but remembered nicolas [comment](https://github.com/symfony/symfony/pull/57683#discussion_r1673680083), about named arguments, so I made it like this 😅

Reply:
I think it's fine to use it in test classes. we avoid using it in code classes, yes

Reply:
Got it